add css to p2code,php

WChihWen opened the pull request:
add css to p2code,php

WChihWen committed:
add css to p2code,php

add css to p2code,php

Ksynlsn commented on css/styles.css:
```diff
@@ -0,0 +1,99 @@
+    font-weight: bold;
+}
+
+.fieldset {
```
This is one small thing I wanted to change: There is a class of "fieldset" on the fieldset element, which is unnecessary.

Ksynlsn commented on css/styles.css:
```diff
@@ -0,0 +1,99 @@
+    color:rgb(160, 59, 46);
+    margin-bottom: 10px;
+}
+footer {
```
Regarding footer CSS: We can remove 'clear: both', as we are not floating any elements.
I think we also need to remove 'position: fixed' and its offset values and position our footer another way. This is what caused the error Kyyrah mentioned in class where the 'Converted Temp' div displayed over the footer links. 

Note: It looks like a position of "relative" was not set on the form element- if we make that change first, the footer styling (with position: fixed) may work as was intended.

Ksynlsn commented on p2code.php:
```diff
@@ -1,92 +1,91 @@
-
-<h1>Temperature Converter</h1>
-<form action="" method="post">
-    <fieldset class="fieldset">
```
Here, I'd remove the class="fieldset" on the fieldset element and simply use the element name for styling purposes

WChihWen committed:
update css

1. styles.css: remove a class of "fieldset" on the fieldset element
2. styles.css: remove 'clear: both'  and position: fixed'
3. p2code.php: the class="fieldset" on the fieldset element
